specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-05-01-preview/BillingStatistics.json

```diff
@@ -0,0 +1,233 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/billingStatistics": {
+      "get": {
+        "x-ms-examples": {
+          "Get all billing statistics for a Sentinel workspace.": {
+            "$ref": "./examples/billingStatistics/GetAllBillingStatistics.json"
+          }
+        },
+        "tags": [
+          "billingStatistics"
+        ],
+        "description": "Gets all billing statistics for a Sentinel workspace.",
+        "operationId": "BillingStatistics_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BillingStatisticList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>null [](http://example.com/codeflow?start=26&length=4)

Why do you want to change this? Because its not really pageable? What if you want it to be pageable in the future? Won't you just add this back? Why churn?

Reply:
Hi Tim! 
It is not pageable now, but we may wish for it to be in the future.
Given this- is standard practice to include the "x-ms-pageable" & set "nextLinkName" to null OR to not include the 'x-ms-pageable' at this time?
Thanks :)!

Reply:
I think you can just include the next link in the BillingStatisticList and dont return a value untill you implement it .. 


Currently this does not really make any sense since you are calling it out as pageable but not actually explaining how to do the paging.

Reply:
I've added nextLink in the BillingStatisticList